python/src/tests/test_consume_message.py

```diff
@@ -28,6 +28,7 @@ def test_read_sqs_message_fetches_messages_from_the_correct_queue(self):
         with mock.patch("boto3.resource") as m, stubber:
             m.return_value = sqs
             _read_sqs_message()
+            stubber.assert_no_pending_responses()
```

Comment:
What are those and why are they necessary?

Reply:
this ensures that all the calls that were added with `add_response` are actually called. Makes sure that there are exactly as many aws calls performed as expected.